ivy_tests/test_docstrings.py

```diff
@@ -30,8 +33,28 @@ def test_docstrings(backend):
         "namedtuple",
         "DType",
         "Dtype",
+        "multinomial",
+        "num_cpu_cores",
+        "get_all_ivy_arrays_on_dev",
+        "num_ivy_arrays_on_dev",
+        "function_unsupported_dtypes",
+        "randint",
+        "unique_counts",
+        "unique_all",
     ]
 
+    skip_list_temp = [
```

Comment:
Why are these being skipped? Maybe you can add in a comment at the top of the function explaining why.

Reply:
yeah, these are skipped because of an issue in their implementation, I will comment these , thanks!